docs/troubleshoot/launcher/launcher-troubleshoot.md

```diff
@@ -49,3 +49,30 @@ CONFIG=/path/to/zowe.yaml
 /*
 ```
 :::
+
+## Troubleshooting Port Validation
+
+On startup Zowe will use `netstat` to check whether the ports required by its enabled components are available.
+
+Under configuration of dual stack networking this may fail, with messages similar to the following:
+
+```
+Netstat test fail with exit code 16 (EZZ2376I Could not determine TCPIPjobname, using default of 'INET'
+...
+EZZ2377I Could not establish affinity with INET (1011/11B3005A) - can not provide the requested option information)
+```
+
+**Follow these steps:**
+
+Update the Zowe configuration property `zowe.network.server.validatePortFree` to value `false`.
+
+```
+zowe: 
+  network:
+    server:
+      validatePortFree: false
+```
+If you are using PARMLIB configurations
```

Comment:
What does it mean PARMLIB configuration here?
Is it DD `STDENV` (environment variable definitions) in Zowe Launcher STC? 

The term "PARMLIB" is usually referring to the config entry - yaml saved in dataset in format of `PARMLIB(DATASET.TO.YAML(MYCONFIG))`

Reply:
Hi @Martin-Zeithaml ,
Yes that's what we wanted to document.  For the `ZWESLSTC` config this can be either in `.yaml` or a PDS member
```
CONFIG=/SYSTEM/var/lpp/zowe/v2/zowe.yaml
```
or
```
CONFIG=PARMLIB(LPAR.PARMLIB(ZWECFG)
```
as documented in https://docs.zowe.org/stable/user-guide/configmgr-using/#parmlib-support.
For the parmlib instead of the indented `zowe.yaml` format to set property keys and values, we wanted to describe the line that needs setting.  
```
ZWE_NETWORK_VALIDATE_PORT_FREE=false
```
I'll think of a rewrite though as you're correct it's not as clear as it could be.  Thank you for the feedback !